### DISCUSSION TITLE
Should the UPower module be merged or renamed?

### DISCUSSION THREAD

**Discussion by JakeStanger** [JakeStanger/ironbar#442] [Polls]
Currently I've got a bit of a gripe with the UPower module, in that it is named after the underlying service that provides the data. 

In all other cases, I try and abstract this a bit to make it obvious what the module is (for example, `music` can connect to MPRIS or MPD). The idea is that this makes the list slighlty more intuitive and requires less of a learning curve to get up and running.

My other issue with it is that it provides battery, which is system information. This could be part of the `sys_info` module, which again could make things just a little simpler.

I'm not rushing to make any changes, but I would like some feedback on this one to see what other people think. Depending on the results, I may make changes in the future.

Any thoughts or other ideas, please do stick them in a comment below.

**Comment by yavko** [comment]
Maybe just `battery`?

**Reply by JakeStanger** [reply]
`battery` it is. Created a tracking issue: https://github.com/JakeStanger/ironbar/issues/934
